nitrite-spatial/src/main/java/org/locationtech/jts/util/GeometricShapeFactoryExt.java

```diff
@@ -0,0 +1,45 @@
+package org.locationtech.jts.util;
```

Comment:
Lets use nitrite package name here.

Reply:
Ah, I thought this needed to be in the same package as the existing `GeometricShapeFactory` in order to access the protected fields, but I see now that I was misremembering my Java inheritance visibility rules! 😄 I did a quick refactor and it works just fine. Pushing an update...